### PR TITLE
Update klarna-customer-experience.md

### PR DESCRIPTION
**Pull request by katkinssss**
## Purpose of this pull request

Screenshots and text changed a little bit due to changes in logo, iframe and wording

whatsnew
Changes to [Klarna Storefront Experience](https://docs.magento.com/user-guide/payment/klarna-customer-experience.html) to capture updates for the Klarna logo, iframe, and labeling.

### CODE REVIEW

File: src/payment/klarna-customer-experience.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ title: Klarna Storefront Experience
 
 Klarna appears as a payment method during checkout, if it is enabled in the configuration. When data sharing is used, Klarna can instantly prequalify the customer and present the available solutions.
 
-![Pay over time with Klarna]({% link images/images/storefront-cart-payment-method-klarna2.png %}){: .zoom}
-_Storefront - Pay over time with Klarna_
+![Pay later with Klarna]({% link images/images/storefront-cart-payment-method-klarna_ot.png %}){: .zoom}
```
Comment (dobooth) [inline src/payment/klarna-customer-experience.md:7]: If you are changing images, please delete the original image (If it is not used somewhere else).

**Review by dobooth — CHANGES_REQUESTED** [review]

File: Gemfile.lock [hunk 2]
```diff
@@ -181,4 +181,4 @@ DEPENDENCIES
   whatsup_github
 
 BUNDLED WITH
-   2.2.15
+   2.2.16
```
Comment (dshevtsov) [inline Gemfile.lock:184]: Changes to this file are not allowed. Please revert this change.

**Review by dshevtsov — CHANGES_REQUESTED** [review]
`Gemfile.lock` does not relate to content. Please revert any changes there.

File: src/payment/klarna-customer-experience.md (hunk 1 above)
Comment (katkinssss) [reply src/payment/klarna-customer-experience.md:7]: Done, please check again

File: Gemfile.lock (hunk 2 above)
Comment (katkinssss) [reply Gemfile.lock]: done

**Review by dshevtsov — APPROVED** [review]
Approving my part

**Review by leslietilling — APPROVED** [review]

File: src/payment/klarna-customer-experience.md (hunk 1 above)
Comment (katkinssss) [reply src/payment/klarna-customer-experience.md:7]: Hi again! Is there anything left from my side? @dobooth

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1211`.
